bodyparsers/msgpack/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@loopback/bodyparser-msgpack",
+  "version": "0.1.0",
+  "description": "Body parser to handle MessagePack requests in LoopBack 4.",
+  "keywords": [
+    "loopback-extension",
+    "loopback-bodyparser",
+    "loopback"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=10.16"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run clean && npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "bodyparsers/msgpack"
+  },
+  "author": "IBM Corp.",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "dependencies": {
+    "@loopback/core": "^2.9.4",
+    "@loopback/rest": "^6.1.0",
```

Comment:
We are in the process of refactoring extensions to use peer dependencies for core packages, see https://github.com/strongloop/loopback-next/pull/5959

Can you please follow the new convention in your new component?